app/femr/business/services/system/MissionTripService.java

```diff
@@ -155,6 +155,9 @@ public MissionTripService(IRepository<IMissionCity> missionCityRepository,
     @Override
     public Optional<IMissionTrip> retrieveCurrentMissionTrip(int userId) {
 
+        //TODO should this be moved to just be identified whenever a CurrentUser is retrieved?
+        // This should be contained in the session service??
+
```

Comment:
I agree with @kevinzurek. It does make sense that maybe this function could be in the sessionService, but I think keeping it in the MissionTripService we have the ability to get the Mission Trip for any user. 

Maybe instead we could rename this function to `retrieveMissionTripForUser(int userId)` to be more clear on what it is doing?